test/unit_tests/README.md

```diff
@@ -0,0 +1,37 @@
+# How to build the test/capgen_test (on hera)
+
+## Quick start:
+```
+cd test/capgen_test
+mkdir build
+cd build
+cmake ..
+make
+./test_host
+```
+
+The command to run ccpp_capgen.py is:
+
+`/scratch1/BMC/gmtb/Julie.Schramm/NEMSfv3gfs/ccpp/framework/scripts/ccpp_capgen.py \
```

Comment:
Should we use relative paths to the top-level directory `NEMSfv3gfs` instead of absolute paths?